Blockzilla/SettingsViewController.swift

```diff
@@ -562,8 +575,8 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     }
 
     private func updateSafariEnabledState() {
-        guard let safariIndex = toggles.first(where: { $1.setting ==  SettingsToggle.safari})?.key,
-            let safariToggle = toggles[safariIndex]?.toggle else { return }
+        guard let index = Section.getSections().index(where: { $0 == Section.integration }),
```

Comment:
We will likely need a similar function for the searchSuggestions toggle since users will be able to enable it from the URL bar. Perhaps this should be made generic and we pass in which toggle we want to update.

Reply:
I worked on this a bit. I don't know if generalizing it is the right idea. What makes the safari option special is that it actually looks at safaris settings in the phones Settings before proceeding. Also, we don't need this for SearchSuggestions yet, so shouldn't we change it if and when we do? We already update the preferences internally elsewhere.

Reply:
Fair enough, let's just be sure to add that functionality when we add the ability to toggle it outside of settings! :)